### DISCUSSION TITLE
RX-V475 works

### DISCUSSION THREAD

**Discussion by FulcoRH** [mvdwetering/yamaha_ynca#226] [General]
On the RX-475 it will works.
Only my question how to integrate now the media player to use the RX-V475.
What is missing in the manual where place the code for the remote control, which yaml to use.

**Comment by mvdwetering** [comment]
I am not sure what you want to achieve could explain a bit more?

**Comment by FulcoRH** [comment]
Want to use the media player in HA and play/stream the music on the RX-V475. I missing now the link to play/stream it

**Comment by mvdwetering** [comment]
At this moment browsing music and then starting music is not supported by this integration. I have looked into that recently for another person requesting it, but have not found a way yet to have it work reliably. 

Note that starting playback from a URL will never work as the procotol I am using does not support that, only start music from the mediabrowser on the receiver will be possible.

As an alternative you could try the DLNA Digital Media Renderer integration in HA. It does not work well for my receiver, but yours is newer so maybe it works better.

**Comment by FulcoRH** [comment]
Michel thanks.
The DLNA renderer works. Only the server is down I have to hotswap a HDD of 1T.
But only the radio part of HA doesn't stream the sound to the Reciever only to the webbrowser.
